packages/web-components/src/components/content-group/content-group-heading.ts

```diff
@@ -28,12 +29,22 @@ class DDSContentGroupHeading extends StableSelectorMixin(LitElement) {
   @property({ reflect: true })
   slot = 'heading';
 
-  connectedCallback() {
-    super.connectedCallback();
+  /**
+   * Render the heading tag into the light DOM of this component.
+   */
+  protected _renderHeading() {
+    const template = document.createElement('template');
+    template.innerHTML = `<h3>${this.innerHTML.trim()}</h3>`;
+    const heading = template.content.firstChild;
+    render(html`${heading}`, this);
+  }
+
+  firstUpdated() {
+    this._renderHeading();
```

Comment:
I think that can work. Will just need to check if the `<h3>` was already added or not and handle accordingly. Will try that.

Reply:
On second thought, I don' think we need to worry about that. In case target makes adjustments before this runs, then we'll sweep up their adjustments, wrapping them in the `<h3>` and writing the result to the light DOM. In case target makes adjustments after this runs, it'll get reflected, b/c they are changing light DOM that all goes to the default slot. Or am I missing something?

[Implementations like `<dds-content-block-heading>`](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1/packages/web-components/src/components/content-block/content-block-heading.ts#L52) are likely problematic for Target as the light DOM slotted content is copied, and then ignored. Updates after the componen is initialized are ignored.